pkg/api/helpers.go

```diff
@@ -178,3 +179,53 @@ func LoadBalancerStatusDeepCopy(lb *LoadBalancerStatus) *LoadBalancerStatus {
 	}
 	return c
 }
+
+// GetAccessModesAsString returns a string representation of an array of access modes.
+// modes, when present, are always in the same order: RWO,ROX,RWX.
+func GetAccessModesAsString(modes []PersistentVolumeAccessMode) string {
+	modesAsString := ""
+
+	if contains(modes, ReadWriteOnce) {
+		appendAccessMode(&modesAsString, "RWO")
```

Comment:
Why not keep a map[string]string that maps api.PersistentVolumeAccessMode -> short-string and vice-versa, then just iterate here and below.  That way you only spell out the full list one time.  You just KNOW that one day there will be a new mode and you will have to hunt down 75 places you hardcoded these.